packages/neptune-extension/src/labextension/components/ToolbarButton.less

```diff
@@ -1,6 +1,10 @@
 .n-toolbar-button {
   min-width: 27px !important;
 
+  &__icon {
```

Comment:
Would be great to add comment why !important is needed.
I assume someone can remove it in the future and do not notice it broke Lab.